### PR TITLE
Draft Gradle 4.4 release notes preamble

### PR DESCRIPTION
**Pull request by eriwen**

### CODE REVIEW

**Comment by marcospereira** [comment]
Hi @eriwen, hope I'm not trying to sell this too hard, but I think it is also worth noticing the improvements in supporting Play 2.6 and Scala 2.12.

Gradle 4.4 will have fixes for `gradle runPlayBinary` which now works with Play 2.6 and fully supports the combination of Play 2.6 and Scala 2.12; `gradle dist` now generates the correct script to start the built application and finally, there are improvements on Play plugin documentation.

You can see #3018, #3314 and #3315 for more details.

**Comment by eriwen** [comment]
Not selling too hard at all @marcospereira! This is a great suggestion, I just didn't see it in the release notes. I will add a section on it — thanks!

**Comment by eriwen** [comment]
Thank you again @marcospereira. This is merged into `release`

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -1,73 +1,83 @@
...
-### Plugin repositories enhancements
+### Improved Play 2.6 support
 
-Plugin repositories declared in a settings script can now have custom names:
+This version of Gradle improves the `runPlayBinary` task to work with Play 2.6.
 
+* The combination of Play 2.6 and Scala 2.12 should now have full support
+* Play's `dist` task fixes the generated start script
+
+You can read more in the improved [Play plugin user guide chapter](userguide/play_plugin.html). Special thanks to [Marcos Pereira](https://github.com/marcospereira) for extraordinary contributions here.
```
Comment (marcospereira) [inline subprojects/docs/src/docs/release/notes.md]: ❤️

**Comment by marcospereira** [comment]
Thank you so much, @eriwen. I don't know how the release process works for Gradle, but I just noticed that #3315 changes were not cherry-picked to `release` branch.

**Comment by eriwen** [comment]
Thanks for the heads up @marcospereira. I've asked @blindpirate to do so.

**Pull request closed without merging**